src/mame/taito/taito_f3.h

```diff
@@ -369,6 +369,8 @@ class taito_f3_state : public driver_device
 	void dpix_1_sprite(u32 s_pix);
 	void dpix_bg(u32 bgcolor);
 	void init_alpha_blend_func();
+	void get_pixmap_pointer(const int skip_layer_num, const f3_playfield_line_inf **line_t, const int y);
+	void culc_pixmap_pointer(const int skip_layer_num);
```

Comment:
Please don’t use `const` parameters passed by value – compilers are inconsistent about whether the `const` becomes part of the mangled name, and it causes header churn if you need to change it for an implementation change.

Reply:
What is the preferred solution here, `const &` or simply dropping the `const`?

Reply:
Just drop the `const` – it will be passed by value so local modification won’t leak back to the caller.

Reply:
Alright, done! Removed const from pass-by-value params.